### PR TITLE
Add sample for loading data using kafka

### PR DESCRIPTION
**Pull request by LakshanWeerasinghe**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: load-data-from-kafka/main.bal [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import ballerinax/kafka;
+import ballerina/time;
+import load_data_from_kafka.leads;
+import ballerina/uuid;
```
Comment (ShammiL) [inline load-data-from-kafka/main.bal]:
```suggestion
import load_data_from_kafka.leads;

import ballerina/time;
import ballerina/uuid;
import ballerinax/kafka;
```

File: load-data-from-kafka/main.bal [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+public type LeadAnalyticsData record {|
+    string LeadID;
```
Comment (SasinduDilshara) [inline load-data-from-kafka/main.bal]: shall we make the first letter simple

File: load-data-from-kafka/main.bal [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    string FirstName;
+    string LastName;
+    string Email;
+    string Phone;
+    string Company;
+    string Status;
+    string Source;
+    float LeadScore;
```
Comment (SasinduDilshara) [inline load-data-from-kafka/main.bal]: Do we need this Lead prefix, shall we just use, `id, score, owner, createdDate, contactDate`.

File: load-data-from-kafka/persist/model.bal [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import ballerina/persist as _;
+import ballerina/time;
+
+public type LeadAnalyticsData record {|
+    readonly string id;
+    string LeadID;
```
Comment (SasinduDilshara) [inline load-data-from-kafka/persist/model.bal]: shall we use simple letters at the start

File: load-data-from-kafka/Ballerina.toml [hunk 5]
```diff
@@ -0,0 +1,18 @@
+[package]
+org = "lakshanweerasinghe"
```
Comment (pcnfernando) [inline load-data-from-kafka/Ballerina.toml]:
```suggestion
org = "etl"
```

**Comment by pcnfernando** [comment]
Can we rethink of the sample name since kafka is the source in this case.  May be `Kafka-DB-Ingestion`

**Comment by pcnfernando** [comment]
Add readme.md

File: load-data-from-kafka/generated/leads/persist_client.bal [hunk 6]
```diff
@@ -0,0 +1,104 @@
+// AUTO-GENERATED FILE. DO NOT MODIFY.
+// This file is an auto-generated file by Ballerina persistence layer for model.
+// It should not be modified by hand.
+import ballerina/jballerina.java;
```
Comment (pcnfernando) [inline load-data-from-kafka/generated/leads/persist_client.bal]: Let's remove the generated stuff from here and in the readme add the step to `persist generate`

File: load-data-from-kafka/main.bal (hunk 1 above)
Comment (LakshanWeerasinghe) [reply load-data-from-kafka/main.bal]: updated.

File: load-data-from-kafka/persist/model.bal (hunk 4 above)
Comment (LakshanWeerasinghe) [reply load-data-from-kafka/persist/model.bal]: updated.

File: load-data-from-kafka/Ballerina.toml (hunk 5 above)
Comment (LakshanWeerasinghe) [reply load-data-from-kafka/Ballerina.toml]: updated.

File: load-data-from-kafka/main.bal (hunk 3 above)
Comment (LakshanWeerasinghe) [reply load-data-from-kafka/main.bal]: updated.

File: kafka-db-ingestion/docker-compose.yml [hunk 7]
```diff
@@ -0,0 +1,34 @@
+version: '3'
+services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper:latest
```
Comment (pcnfernando) [inline kafka-db-ingestion/docker-compose.yml:4]: shall we stick to specific version for all dependent services

File: kafka-db-ingestion/mysql-init.sql [hunk 8]
```diff
@@ -0,0 +1,25 @@
+CREATE DATABASE leads;
```
Comment (pcnfernando) [inline kafka-db-ingestion/mysql-init.sql:1]: Let's use the docker compose to have the DB name

File: kafka-db-ingestion/persist/model.bal [hunk 9]
```diff
@@ -0,0 +1,21 @@
+import ballerina/persist as _;
+
+public type LeadAnalyticsData record {|
```
Comment (pcnfernando) [inline kafka-db-ingestion/persist/model.bal:3]:
```suggestion
type LeadAnalyticsData record {|
```

**Review by pcnfernando — APPROVED** [review]
Suggestions will be handled separately. Hence approving

**Pull request merged**